🚀 #128 puck a panch

ivardeviking opened the pull request.

Blamant committed:
Working PAP

Blamant committed:
m249 m4a1 and mp5 pap'ed

Blamant committed:
new sounds added and bullet sprites

Blamant committed:
All weapons sounds, sprites, bullets added. Also Jukebox added. Also bug fixes and buy/NEM sounds added

Blamant committed:
Merge branch 'alpha' into #128-puck-a-panch

Blamant committed:
Electrical humming added and bugs fixed

Commit:
script path fix

ivardeviking commented:
Enkele bugs:
- ~~Node path aangepast om het te laten werken in level 2~~ Fixed with 90f5d0e
- Electrical humming inderdaad na 30 seconden uit.
- enkele sprites te groot waardoor het buiten de weapon boxes in de HUD valt

Verbeteringen:
- wanneer je je wapen panch a pucked, dan wegdoet en opnieuw koopt blijft hij gepanch a pucked.
- Ik wilde een deur openen en panch a pucked perongeluk mijn wapen. Misschien een idee om hem op hold te zetten in plaats van pressed.
- Total ammo van verschillende nieuwe wapens is nog wat weinig (UMP bijvoorbeeld maar 100 kogels)

JoJ3o commented:
> wanneer je je wapen panch a pucked, dan wegdoet en opnieuw koopt blijft hij gepanch a pucked.

 Het systeem om een wapen opnieuw te kopen doet het sowieso nog niet, dus daar moeten we volgende sprint naar kijken. (Het is Juke a Box trouwens)

> Ik wilde een deur openen en panch a pucked perongeluk mijn wapen. Misschien een idee om hem op hold te zetten in plaats van pressed.

Gewoon de jukebox niet naast een deur zetten zou dit al moeten fixen, maar misschien wel leuk om een 5 seconden hold te houden om het wat meer risico te geven om je wapen te upgraden.

Blamant committed:
Multiple bug fixes for pull request

- Fixed sprite size
- Adjusted interface to accommodate for large weapon sprites
- Added broken light to the Jukebox
- Added light2d to the laser bullets
- Fixed humming sound not looping

Commit:
fix path (again)

JoJ3o committed:
Change perk and pap positions

JoJ3o committed:
Little Improvements

- Move tombstone behind door
- Give healthbar a number
- Reorder weapon slots
- Give weapon slots a number
- Replace weapons and perks
- Reduce regen time
